pipeline-runner/R/qc-4-filter_gene_umi_outlier.R

```diff
@@ -1,19 +1,27 @@
 # STEP 4. Number of genes vs UMIs filter
+# This filter focuses on filter cells that are far from the behaviour of the relationship between the number of genes (it measures the number of
+# genes in a cell that has at least one count) and the number of UMIs/molecules (the total number of counts in a cell).
+#' @description Filters seurat object based on classifier filter
```

Comment:
If you feel like it (feel free to skip) -  check that `roxygen2` builds the documentation correctly `ctrl+shift+D`. `roxygen2` format is typically:

```R
#' Title
#'
#' Description
#'
#' Details
#'
#' @param blah
#' @export
#' @return what is returned
```

You can generate a skeleton for a function by putting cursor in function and typing `ctrl+alt+r`